lib/pengine/pe_output.c

```diff
@@ -1284,6 +1284,109 @@ pe__node_attribute_html(pcmk__output_t *out, va_list args) {
     return pcmk_rc_ok;
 }
 
+PCMK__OUTPUT_ARGS("node-and-op", "pe_working_set_t *", "xmlNode *")
+int
+pe__node_and_op(pcmk__output_t *out, va_list args) {
+    pe_working_set_t *data_set = va_arg(args, pe_working_set_t *);
+    xmlNode *xml_op = va_arg(args, xmlNode *);
+
+    pe_resource_t *rsc = NULL;
+    gchar *node_str = NULL;
+    char *last_change_str = NULL;
+
+    const char *op_rsc = crm_element_value(xml_op, "resource");
+    const char *status_s = crm_element_value(xml_op, XML_LRM_ATTR_OPSTATUS);
+    const char *op_key = crm_element_value(xml_op, XML_LRM_ATTR_TASK_KEY);
+    int status = crm_parse_int(status_s, "0");
+    time_t last_change = 0;
+
+    rsc = pe_find_resource(data_set->resources, op_rsc);
+
+    if (rsc) {
+        pe_node_t *node = pe__current_node(rsc);
+        const char *target_role = g_hash_table_lookup(rsc->meta, XML_RSC_ATTR_TARGET_ROLE);
+        int opts = pe_print_printf | pe_print_rsconly | pe_print_suppres_nl | pe_print_pending;
```

Comment:
Don't need pe_print_printf or pe_print_suppres_nl with formatted output

Reply:
Fixed.  This is the danger of just moving code over from an old function to a new one without really studying it.